.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: ruby
 
+rbenv:
```

Comment:
We go to a build matrix scheme in order to run the non-Dockerized tests across Ruby versions.